web/src/app/components/bar-chart/bar-chart.component.ts

```diff
@@ -0,0 +1,341 @@
+import { Component, ElementRef, Input, OnChanges, SimpleChanges, ViewChild, AbstractType} from '@angular/core';
+import { formatDate } from '@angular/common'
+import * as d3 from 'd3';
+
+import { step189_2020 } from '../../../proto/step189_2020';
+
+/**
+ * BarChartData includes all data used by the bar chart.
+ */
+interface BarChartData {
```

Comment:
Razvan left a comment on this for me as well, but essentially this name can be misleading since "BarChartData" implies that it contains _all_ the data rather than just one singular bar. He suggested I rename to `Item`